config.example.json

```diff
@@ -41,6 +41,16 @@
       "data": {
         "url": "https://example.com/webhook"
       }
+    },
+    {
+      "id": "unique-id-whatsapp",
+      "type": "whatsapp",
+      "data": {
+        "recipients": ["08123456789"],
```

Comment:
Phone number recipient is using local code (start with 0) instead of country code (ex: 62, 65, 1)?

Reply:
fixed